### PR TITLE
[FlagGems Operator Development Competition] Add logaddexp operator

### PR DESCRIPTION
**Pull request by Potato-wll**
Add `torch.logaddexp` operator implementation using Triton, along with unit tests and performance benchmark.

`logaddexp(x, y)` computes `log(exp(x) + exp(y))` in a numerically stable way using the formula:
`max(x, y) + log(1 + exp(min(x, y) - max(x, y)))`.

### Operator Implementation
- **`src/flag_gems/ops/logaddexp.py`** (new): Triton kernel implementation using `pointwise_dynamic`, with float16/bfloat16 upcast to float32 for numerical stability.

### Registration
- **`src/flag_gems/ops/__init__.py`**: Import and export `logaddexp`.
- **`src/flag_gems/__init__.py`**: Register `logaddexp` to aten backend.

### Unit Tests
- **`tests/test_binary_pointwise_ops.py`**: Added 7 test functions (45 test cases total):
  - `test_accuracy_logaddexp` — standard accuracy across multiple shapes × dtypes
  - `test_accuracy_logaddexp_boundary_zeros` — zero-value inputs
  - `test_accuracy_logaddexp_boundary_negatives` — all-negative inputs
  - `test_accuracy_logaddexp_boundary_extremes` — inf / -inf extreme values
  - `test_accuracy_logaddexp_broadcast` — broadcast shape compatibility
  - `test_accuracy_logaddexp_empty_tensor` — empty tensor handling
  - `test_accuracy_logaddexp_large_diff` — numerical stability with large input differences

### Performance Benchmark
- **`benchmark/test_binary_pointwise_perf.py`**: Added `logaddexp` to binary pointwise benchmark.

## Test Results

```shell
pytest -m logaddexp
```
image

```shell
pytest benchmark/test_binary_pointwise_perf.py -m logaddexp -s
```
image

image
image

## Checklist

- [x] Operator implementation (`src/flag_gems/ops/logaddexp.py`)
- [x] Operator registration (`__init__.py`)
- [x] Unit tests with `@pytest.mark.logaddexp` decorator
- [x] Boundary cases covered (zeros, negatives, inf, empty tensor, large diff)
- [x] Performance benchmark added
- [x] All 45 test cases passed
- [x] Pre-commit format check passed

### CODE REVIEW

**Comment by you-and-you** [comment]
Resolve merge conflicts

File: tests/test_binary_pointwise_ops.py [hunk 1]
```diff
@@ -2260,37 +2222,130 @@ def test_accuracy_addcdiv(shape, dtype):
...
+    gems_assert_close(res_out, ref_out, dtype)
+
+
+@pytest.mark.logaddexp
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_accuracy_logaddexp_boundary_extremes(dtype):
+    shape = (256,)
+    inp1 = torch.randn(shape, dtype=dtype, device=flag_gems.device)
+    inp2 = torch.randn(shape, dtype=dtype, device=flag_gems.device)
+    # 混入极值：一个 inf/−inf 配一个有限值
```
Comment (you-and-you) [inline tests/test_binary_pointwise_ops.py]: Remove this Chinese comment.

**Comment by douxetpur** [comment]
Hi, thank you for your participation in the FlagOS Open Computing Competition 🙏

After review, we have chosen to move forward with a different implementation for this operator. As a result, this PR will be closed.

We appreciate your time and contribution, and hope to see more of your submissions in the future.
